playbooks/roles/thoth-kebechet/tasks/main.yaml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
We can completely remove this file, both projects/namespaces are required (aka assume they are created before you deploy kebechet) and need to be set via a extra var of the playbook, see https://github.com/thoth-station/kebechet/pull/11/files#diff-88b99bb28683bd5b7e3a204826ead112R17